webapp/channels/src/packages/mattermost-redux/src/selectors/entities/posts.ts

```diff
@@ -102,24 +101,6 @@ export function getPostIdsInCurrentChannel(state: GlobalState): Array<Post['id']
     return getPostIdsInChannel(state, state.entities.channels.currentChannelId);
 }
 
-export type PostWithFormatData = Post & {
-    isFirstReply: boolean;
-    isLastReply: boolean;
-    previousPostIsComment: boolean;
-    commentedOnPost?: Post;
-    consecutivePostByUser: boolean;
-    replyCount: number;
-    isCommentMention: boolean;
-    highlight: boolean;
-};
-
-// getPostsInCurrentChannel returns the posts loaded at the bottom of the channel. It does not include older posts
-// such as those loaded by viewing a thread or a permalink.
-export const getPostsInCurrentChannel: (state: GlobalState) => PostWithFormatData[] | undefined | null = (() => {
```

Comment:
This selector still exists for use by `addVisibleUsersInCurrentChannelToStatusPoll`, but I replaced the code for `makeGetPostsInChannel` with it below since we didn't need it to get posts for arbitrary channels anymore